src/css/spectrum/spectrum.css

```diff
@@ -13,7 +13,7 @@ License: MIT
     *display: inline;
     *zoom: 1;
     /* https://github.com/bgrins/spectrum/issues/40 */
-    z-index: 9999994;
+    z-index: 20000;
```

Comment:
![tooltip](https://cloud.githubusercontent.com/assets/2543511/13466652/391d8d22-e09b-11e5-90f0-3c99f4e241d6.png)
Here's what we see at the moment. The informative tooltip for the red selector is cut in half.

It's neither practical nor important. There's a title "tooltip" on the palette handled by Spectrum.